code/modules/admin/IsBanned.dm

```diff
@@ -1,20 +1,45 @@
 //Blocks an attempt to connect before even creating our client datum thing.
 
+GLOBAL_LIST_EMPTY(isbanned_key_floodcheck)
+GLOBAL_LIST_EMPTY(isbanned_cid_floodcheck)
+GLOBAL_LIST_EMPTY(isbanned_ip_floodcheck)
+
+#define MAX_CONCURRENT_IP_CONNECTIONS 2

```

Comment:
"my brother needs to connect at the same time" situations

Reply:
lmao